acq_stat_reports/scripts/acq_stat_reports.py

```diff
@@ -0,0 +1,140 @@
+"""
+Generate acquisition statistics report.
+"""
+
+import argparse
+import json
+import os
+from pathlib import Path
+
+import ska_report_ranges
+from astropy import units as u
+from cxotime import CxoTime
+from ska_helpers import logging
+
+from acq_stat_reports import config as conf
+from acq_stat_reports import get_acq_info, get_data, make_acq_plots, make_html
+
+SKA = Path(os.environ["SKA"])
+
+
+logger = logging.basic_logger("acq_stat_reports", level="INFO")
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--datadir",
+        default=Path(SKA / "data" / "acq_stat_reports"),
+        help="Output directory (default is $SKA/data/acq_stat_reports)",
+        type=Path,
+    )
+    parser.add_argument(
+        "--webdir",
+        default=Path(SKA / "www" / "ASPECT" / "acq_stat_reports"),
+        help="Output directory (default is $SKA/www/ASPECT/acq_stat_reports)",
+        type=Path,
+        dest="output_dir",
+    )
+    parser.add_argument(
+        "--url",
+        help="URL root of the deployed page (default is /mta/ASPECT/acq_stat_reports/)",
+        default="/mta/ASPECT/acq_stat_reports/",
+    )
+    parser.add_argument(
+        "--start_time", help="Start time (default is NOW)", default=None
+    )
+    parser.add_argument(
+        "--days_back",
+        help="How many days to look back (default is 30)",
+        default=30,
+        type=int,
+    )
+    parser.add_argument(
+        "--log-level",
+        default="INFO",
+        choices=[
+            "DEBUG",
+            "INFO",
+            "WARNING",
+            "ERROR",
+            "CRITICAL",
+            "debug",
+            "info",
+            "warning",
+            "error",
+            "critical",
+        ],
+        help="Verbosity (DEBUG, INFO, WARNING, ERROR, CRITICAL)",
+    )
+    return parser
+
+
+def main(sys_args=None):
+    import matplotlib
+
+    matplotlib.use("agg")
```

Comment:
Now that this is a dedicated script you can put this at the top level, which makes the dependencies a bit more obvious.

Reply:
done